models/pattern/core/error.go

```diff
@@ -0,0 +1,13 @@
+package core
+
+import (
+	"github.com/layer5io/meshkit/errors"
+)
+
+const (
+	ErrGetK8sComponentsCode = "10000"
```

Comment:
Should this be statically set or should the error code utility determine the appropriate #?, @DelusionalOptimist

Reply:
@Revolyssup fyi - https://docs.meshery.io/project/contributing-error.... "replace_me" instead?

Reply:
will make the change